### PR TITLE
Save File

### PR DESCRIPTION
**Pull request by vritant24**
- Added code to save file
- set up switching of versions

### CODE REVIEW

**Comment by drewatk** [comment]
Ready for review?

File: app/git-abs/git/branch-name.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import branchName from "current-git-branch";
```
Comment (drewatk) [inline app/git-abs/git/branch-name.js]: Looks like this package relies on you having git installed, which I'd rather we not

Comment (drewatk) [reply app/git-abs/git/branch-name.js]: nodegit doesn't have this?

Comment (vritant24) [reply app/git-abs/git/branch-name.js]: It does. I'll fix it in the next sprint

**Pull request merged**